tests/fixtures/create/resultingDocument.json

```diff
@@ -34,7 +34,7 @@
   },
   "methodMetadata": {
     "published": true,
-    "recoveryCommitment": "EiAyVInmF_nzpaefyWj6BoNGdY_9FRNef-WI3crrK6vIyA",
-    "updateCommitment": "EiCjkDGDDOnuZq4Kiw2icudIN95boDMrGdYx4pz8bBt2bw"
+    "recoveryCommitment": "EiAzN_y4WBLpZsjRXg8GbpRtXAhddATz-vmq3AxMOV2Huw",
```

Comment:
is this casing correct?

Reply:
Ya, everything in the resulting document should be camel case, unless my understanding is wrong.

Reply:
This is a resolver response (external data)... this is exactly my point.